build.gradle

```diff
@@ -1837,6 +1837,8 @@ project(':storage') {
     testImplementation project(':clients').sourceSets.test.output
     testImplementation project(':core')
     testImplementation project(':core').sourceSets.test.output
+    testImplementation project(':server')
+    testImplementation project(':server').sourceSets.test.output
```

Comment:
Tests not required in `storage` module - only main classes. Line 1841 can be removed.